### PR TITLE
Solved Resources validation Lint errors.

### PR DESCRIPTION
**Pull request by pandilladivya**
Removed Android Resources validation Lint Issue..

### CODE REVIEW

**Comment by top-gun007** [comment]
hi @pandilladivya it might be great if you do the following

- squash your commits into one if its a single issue.

- use meaningful comments on your commits. 

I was asked the same when I sent my PR.

**Comment by pandilladivya** [comment]
@top-gun007  

I've squashed the commits....I hope this should be it :)

Thank you for the review!

**Comment by sandarumk** [comment]
Remove the .iml file from the PR. There are a certain set of files which you shouldn't commit in Android project. A quick Google search will let you know what should be there and what shouldn't be there. Change the PR accordingly.

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java [hunk 1]
```diff
@@ -83,16 +115,34 @@ public Object getGroup(int groupPosition) {
         return this.dataHeader.get(groupPosition);
     }
 
+    /**
+     * Returns group count.
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java]: Follow the format

File: app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java [hunk 2]
```diff
@@ -142,14 +205,21 @@ public void filter(String textEntered)
...
+     */
     public static void prepareListData(Context context,List<String> listDataHeader,HashMap<String, List<String>> listDataChild)
     {
         listDataChild.clear();
         listDataHeader.clear();
         for(int i =0; i<23;i++)
             listDataHeader.add("");
         Collections.copy(listDataHeader,Arrays.asList(context.getResources().getStringArray(R.array.dataheaders)));
-        // Adding child data
+        // adding child data
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryAdapter.java]: Keep the capital one.

**Comment by pandilladivya** [comment]
@sandarumk Sure will do it right away

Thanks for the review..

**Comment by sandarumk** [comment]
Along with the lint issues you have added comments. Adding comments is a separate issue and do not mixed this with it. What you have to include in this PR is only the fixed lint issues. If you want to add the missing source code comments, you can comment on the issue regarding that, pick a module and do that after you are done with the lint issue PR.

**Comment by pandilladivya** [comment]
@sandarumk 

I've made a new [PR](https://github.com/systers/PCSA-Android/pull/288) 

kindly review

Thank You!

**Pull request closed without merging**